### ISSUE TITLE
Not Working

### ISSUE THREAD

**Issue by Koshurhacker** [Deadpool2000/Paybag#4]
Not Working

**Comment by Deadpool2000** [comment]
Yes. Many users reported some issues and I am currently working on it.

**Comment by Deadpool2000** [comment]
Kindly provide screenshot of issue

**Comment by jereborda** [comment]
<img width="538" alt="Captura de pantalla 2024-07-02 a la(s) 7 31 32 p  m" src="https://github.com/Deadpool2000/Paybag/assets/129632109/ac799b9c-23ab-4d0f-8d9d-7689f68c8059">

metasploit do not work
